token/cli/src/config.rs

```diff
@@ -134,34 +134,45 @@ impl<'a> Config<'a> {
 
         let default_keypair = cli_config.keypair_path.clone();
 
-        let default_signer: Arc<dyn Signer> = {
+        let default_signer: Option<Arc<dyn Signer>> = {
             if let Some(owner_path) = matches.value_of("owner") {
                 signer_from_path_with_config(matches, owner_path, "owner", wallet_manager, &config)
+                    .ok()
             } else {
-                signer_from_path_with_config(
+                match signer_from_path_with_config(
                     matches,
                     &default_keypair,
                     "default",
                     wallet_manager,
                     &config,
-                )
+                ) {
+                    Ok(signer) => Some(signer),
+                    Err(e) => {
+                        if std::fs::metadata(&default_keypair).is_ok() {
```

Comment:
Hehe it turned out really simple in the end, nice! I took another look through the `signer_from_path_with_config` code, and this really seems like the best option